sandboxed_containers/upgrading-sandboxed-containers.adoc

```diff
@@ -1,27 +1,27 @@
 :_content-type: ASSEMBLY
-[id="upgrade-sandboxed-containers"]
-= Upgrade {sandboxed-containers-first}
+[id="upgrading-sandboxed-containers"]
+= Upgrading {sandboxed-containers-first}
 include::_attributes/common-attributes.adoc[]
-:context: upgrade-sandboxed-containers
+:context: upgrading-sandboxed-containers
 
 toc::[]
 
 You can upgrade the components of {sandboxed-containers-first} by upgrading {sandboxed-containers-operator} and {sandboxed-containers-first} artifacts.
 
 [id="sandboxed-containers-upgrade-operator"]
-== Upgrade {sandboxed-containers-operator}
+== Upgrading {sandboxed-containers-operator}
 
-You can use Operator Lifecycle Manager (OLM) to manually or automatically upgrade the {sandboxed-containers-operator}. You can select manual or automatic upgrade during the initial deployment. In the context of manual upgrades, the web console shows the available updates that can be installed by the cluster administrator.
+You can use Operator Lifecycle Manager (OLM) to manually or automatically upgrade the {sandboxed-containers-operator}. Select manual or automatic upgrade during the initial deployment. For manual upgrades, the web console shows the available updates that can be installed by the cluster administrator.
 
 [role="_additional-resources"]
 .Additional resources
 * xref:../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Upgrading installed Operators]
 
 [id="sandboxed-containers-upgrade-artifacts"]
-== Upgrade the {sandboxed-containers-first} artifacts
+== Upgrading the {sandboxed-containers-first} artifacts
 
 The {sandboxed-containers-first} artifacts are deployed onto the cluster using {op-system-first} extensions.
 
 //Update the Red Hat CoreOS extension with the updated link once PR is approved for the Understanding section.
 
-The {op-system} extension `sandboxed containers` contains the required components to run Kata Containers such as the Kata containers runtime, the hypervisor QEMU, and other dependencies. The extension is upgraded when upgrading the cluster to a new release of {product-title}.
+The {op-system} extension `sandboxed containers` contains the required components to run Kata Containers, such as the Kata containers runtime, the hypervisor QEMU, and other dependencies. The extension is upgraded when upgrading the cluster to a new release of {product-title}.
```

Comment:
@tshwartz It's not your problem (you have cleaned up a great deal here) but it seems the "upgrading...." section should be a procedure on how to do what the section titles suggest rather than a very short discussion. Maybe that's a future update?